discord/flags.py

```diff
@@ -757,9 +758,7 @@ def messages(self):
 
         .. note::
 
-            As of April 2022 requires opting in explicitly via the developer portal to receive the actual content of the guild messages.
-            Bots in over 100 guilds will need to apply to Discord for verification.
-            See https://support-dev.discord.com/hc/en-us/articles/4404772028055-Message-Content-Access-Deprecation-for-Verified-Bots for more information.
+            :attr:`message_content` is required to receive the actual content of guild messages.
```

Comment:
Should leave this, it's still valid 
```suggestion
            As of April 2022 requires opting in explicitly via the developer portal to receive the actual content of the guild messages.
            Bots in over 100 guilds will need to apply to Discord for verification.
            See https://support-dev.discord.com/hc/en-us/articles/4404772028055-Message-Content-Access-Deprecation-for-Verified-Bots for more information.
```

Reply:
I moved it to the `message_content` attr documentation, you think it should still be noted on the other message intents?

Reply:
I suppose that's fine